### ISSUE TITLE
Uncaught TypeError: AlephBet.Goal is not a constructor

### ISSUE THREAD

**Issue by gingerlime** [Alephbet/alephbet#42]
## Alephbet 0.21.0+

```html
<html>
<head>
<script src="https://unpkg.com/alephbet@0.21.0/dist/alephbet.min.js"></script>
<script type="text/javascript">

payment = new AlephBet.Goal('payment completed');
</script>
</head>
</html>
```

raises `Uncaught TypeError: AlephBet.Goal is not a constructor`

## Alephbet 0.20.0

works fine

```html
<html>
<head>
<script src="https://unpkg.com/alephbet@0.20.0/dist/alephbet.min.js"></script>
<script type="text/javascript">

payment = new AlephBet.Goal('payment completed');
</script>
</head>
</html>
```

**Comment by joker-777** [comment]
This is super weird. When I tested my changes I just use Kenhub to include the script and it worked. I could see `AlephBet` in `window`. I've been trying now to get a [code pen example](https://codepen.io/joker-777-the-bold/pen/GRjXgYJ) to work but without any luck :(

**Comment by gingerlime** [comment]
this was fixed with 0.23 (at least works now with it), i.e.

```html
<html>
<head>
<script src="https://unpkg.com/alephbet@0.23.0/dist/alephbet.min.js"></script>
<script type="text/javascript">

payment = new AlephBet.Goal('payment completed');
</script>
</head>
</html>
```
